.github/workflows/jira-issue-label-added.yaml

```diff
@@ -0,0 +1,55 @@
+name: Jira Issue Label Added
+
+on:
+  workflow_call:
+    inputs:
+      team-name:
+        type: string
+
+jobs:
+  create-issue-in-stoplight-jira:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'jira'
+    steps:
+    - name: Login
+      uses: atlassian/gajira-login@v3
+      env:
+        JIRA_BASE_URL: ${{ secrets.SMARTBEAR_JIRA_BASE_URL }}
+        JIRA_USER_EMAIL: ${{ secrets.SMARTBEAR_JIRA_USER_EMAIL }}
+        JIRA_API_TOKEN: ${{ secrets.SMARTBEAR_JIRA_API_TOKEN }}
+
+    - name: Search
+      id: search
+      uses: tomhjp/gh-action-jira-search@v0.1.0
+      with:
+        jql: 'summary ~ "${{ github.event.repository.name }}#${{ github.event.issue.number }}:" AND project=${{ vars.SMARTBEAR_JIRA_PROJECT }}'
+
+    - name: Log
+      run: echo "Found issue ${{ steps.search.outputs.issue }}"
+
+    - name: Create
+      id: create
+      uses: atlassian/gajira-create@v3
+      if: steps.search.outputs.issue == ''
+      with:
+        project: ${{ vars.SMARTBEAR_JIRA_PROJECT }}
+        issuetype: Task
+        summary: '${{ github.event.repository.name }}#${{ github.event.issue.number }}: ${{ github.event.issue.title }}'
+        description: |
+          *Issue Link:* ${{ github.event.issue.html_url }}
+
+          ${{ github.event.issue.body }}
+        fields: '{"Stoplight Team": ${{ inputs.team-name }}}'
```

Comment:
is this the set of labels?

Reply:
oh its a parameter here!

Reply:
I think this should assign a team to the issue but I'll have to test it out!